### ISSUE TITLE
Fresh install of alerta cli don't work with heartbeats

### ISSUE THREAD

**Issue by skob** [guardian/alerta#331]
```
(venv_alerta)# pip install -U alerta
Requirement already up-to-date: alerta in ./lib/python3.4/site-packages
Requirement already up-to-date: pytz in ./lib/python3.4/site-packages (from alerta)
Requirement already up-to-date: requests in ./lib/python3.4/site-packages (from alerta)
Requirement already up-to-date: argparse in ./lib/python3.4/site-packages (from alerta)
(venv_alerta)#
(venv_alerta)# alerta heartbeats
ORIGIN                       TAGS                       CREATED              LATENCY TIMEOUT SINCE             STATUS
2017-01-04 15:04:21,318 - alertaclient.shell - ERROR - non-empty format string passed to object.__format__
Traceback (most recent call last):
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 1594, in main
    AlertaShell().run()
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 1582, in run
    args.func(args)
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 507, in heartbeats
    status()
TypeError: non-empty format string passed to object.__format__
(venv_alerta)#
(venv_alerta)# alerta version
alerta 4.8.5
alerta client 4.8.6
requests 2.12.4
```

**Comment by satterly** [comment]
I'm having trouble reproducing this. Can you run the same command but with `--debug` option and paste the output here?
```
# alerta --debug heartbeats
```

**Comment by skob** [comment]
```
2017-01-05 08:51:19,233 - alertaclient.shell - DEBUG - Alerta cli version: 4.8.6
2017-01-05 08:51:19,239 - requests.packages.urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): alerta
2017-01-05 08:51:19,243 - requests.packages.urllib3.connectionpool - DEBUG - http://alerta:80 "GET /heartbeats HTTP/1.1" 200 509
2017-01-05 08:51:19,244 - alertaclient.api - DEBUG - Request Headers: {'Accept-Encoding': 'gzip, deflate', 'User-Agent': 'python-requests/2.12.4', 'Accept': '*/*', 'Connection': 'keep-alive'}
2017-01-05 08:51:19,244 - alertaclient.api - DEBUG - Response Headers: {'Vary': 'Origin', 'Connection': 'keep-alive', 'Content-Length': '509', 'Content-Type': 'application/json', 'Date': 'Thu, 05 Jan 2017 08:51:19 GMT', 'Server': 'nginx', 'Access-Control-Allow-Credentials': 'true'}
2017-01-05 08:51:19,255 - alertaclient.api - DEBUG - Response Body: {
  "heartbeats": [
    {
      "createTime": "2017-01-05T08:51:08.482Z",
      "customer": null,
      "href": "http://alerta/heartbeat/1f397c07-d8a0-4d8b-ab67-09abb804bf85",
      "id": "1f397c07-d8a0-4d8b-ab67-09abb804bf85",
      "origin": "riemann/central",
      "receiveTime": "2017-01-05T08:51:08.482Z",
      "tags": [
        "3.1.0"
      ],
      "timeout": 300,
      "type": "Heartbeat"
    }
  ],
  "status": "ok",
  "time": "2017-01-05T08:51:19.243Z",
  "total": 1
}

ORIGIN                       TAGS                       CREATED              LATENCY TIMEOUT SINCE             STATUS
2017-01-05 08:51:19,274 - alertaclient.shell - ERROR - non-empty format string passed to object.__format__
Traceback (most recent call last):
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 1594, in main
    AlertaShell().run()
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 1582, in run
    args.func(args)
  File "/opt/www/venv_alerta/lib64/python3.4/site-packages/alertaclient/shell.py", line 507, in heartbeats
    status()
TypeError: non-empty format string passed to object.__format__
```

**Comment by satterly** [comment]
Thanks for the debug output. That helped a lot. This fix is now available as alerta client version 4.8.7.

**Comment by skob** [comment]
Thank you!
